guides/source/active_record_callbacks.md

```diff
@@ -254,7 +254,11 @@ Halting Execution
 
 As you start registering new callbacks for your models, they will be queued for execution. This queue will include all your model's validations, the registered callbacks, and the database operation to be executed.
 
-The whole callback chain is wrapped in a transaction. If any _before_ callback method returns exactly `false` or raises an exception, the execution chain gets halted and a ROLLBACK is issued; _after_ callbacks can only accomplish that by raising an exception.
+The whole callback chain is wrapped in a transaction. If any callback raises an exception, the execution chain gets halted and a ROLLBACK is issued. To intentionally stop a chain use:
```

Comment:
To avoid doubts, I'd suggest telling explicitly if `throw :abort` [only] stops the chain or stops the chain and causes `ROLLBACK` to be issued.